docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,304 @@
+    :align: center
+    :scale: 50
+    :alt: Enabling Vertiq IO
+
+    Enabling Vertiq IO
+
+After reboot, and with Vertiq IO enabled, you should now see a :blue:`Vertiq IO` submenu in the QGroundControl parameter settings. Adjust the :red:`VERTIQ_BAUD` parameter to match what your modules will be using. In this tutorial we will be using a baud rate of 921600 which is what we recommend.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/vertiq_io_settings.png
+    :align: center
+    :scale: 50
+    :alt: Vertiq IO Submenu
+
+    Vertiq IO Submenu
+
+Now your Vertiq modules must be configured for proper communication with the flight controller.
+
+Configuring Your Vertiq Modules for Use with IFCI and PX4
+========================================================
+To use your Vertiq modules properly with IFCI, your modules must be flashed with a compatible firmware version. Please consult your module's family page to find if your module supports IFCI. Once flashed with appropriate firmware, connect each module **individually** to IQControlCenter and set the :blue:`UART Baud Rate` and the :red:`Module ID`. As stated previously, we recommend that you use a baud rate of 921600. Both of these parameters will cause the motor to reboot when set, so make sure you reconnect to the motor after each one is set. Ensure that each module connected to the flight controller is set to a different, unique module ID. For this tutorial we will be using the module IDs 0, 1, 2, and 3.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/control_center_settings.png
+    :align: center
+    :scale: 50
+    :alt: Control Center Settings
+
+    Setting Baud Rate and Module ID
+
+With the modules set to unique module IDs, and the baud rate set to match the flight controller's, you can now connect your modules to the flight controller. To do this, find the flight controller's serial port configured to run the Vertiq IO module, and connect the TX of the serial port to each module's RX port. Connect the RX of the serial port to each module's TX port. Connect a commond ground between the flight controller and each module.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_wiring.png
+    :align: center
+    :scale: 100
+    :alt: Module Flight Control Wiring
+
+    Module Flight Control Wiring
+
+Integration Setup
+=================
+
+.. warning::
+    Please remove all propellers from any module you plan on testing. Failure to do so can result in harm to you or others around you. Further, please ensure that your module is secured to a stationary platform or surface before attempting to spin it. 
+
+Connect your flight controller and motors as shown in the diagram above. Ensure that both the motors and flight controller are receiving power. Connect the flight controller to qgroundcontrol, navigate to the :red:`parameters menu`, and select the :blue:`Vertiq IO` subsection as shown below.
```

Comment:
You can probably link here to the buzzer controller/status songs, and tell people that modules will play their startup song when powered properly. https://iqmotion.readthedocs.io/en/feature-px4-ifci-tutorial/manual/manual_buzzer_control.html#startup

Reply:
Nice idea. Done